contents/handbook/people/benefits.md

```diff
@@ -52,10 +52,20 @@ As we are fully remote, we provide [all equipment](/handbook/people/spending-mon
 
 If you ever need change of scenery, we offer $200/month budget towards [coworking or café working](/handbook/people/spending-money#work-space), or [WeWork All Access](https://www.wework.com/solutions/wework-all-access). 
 
-### Off-sites and team socials
+> If you are in the US, we have a WeWork discount via Brex. Please message Eltje or Charles to get a code. 
 
-Ideally, we would like to meet up in person at twice a year. The team was able to meet up in Italy in September 2020, but we haven't been able to travel since, so we are planning a virtual off-site in April 2021 instead (and hopefully a real one on Autumn 2021!).
+### Off-sites 
+
+Ideally, we would like to meet up in person at twice a year. The team was able to meet up in Italy in September 2020, and we are heading to Portugal in August 2021.
 
 For any work-related travel, we use [Project Wren](https://www.wren.co/) for carbon offsetting. 
 
+### Team socials
+
+As a global, all-remote team, we don't get many regular opportunities to socialize with each other outside of our bi-annual offsites.
```

Comment:
"Biannual" means "every 2 years", I suppose this should be "semiannual".

Reply:
Biannual can mean both, but I'll change it to make it clear that it's twice a year

Reply:
Not that is matters that much, but this isn't correct - biannual _only_ means twice per year, though sometimes it is confused with biennial (which does mean every 2 years). However, agreed we should minimise for confusion!

Reply:
Oh, correct, I did actually mix this up with bi_ennial_ – sorry for a false alarm 😅